charts/aws-ebs-csi-driver/templates/clusterrole-attacher.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 rules:
   - apiGroups: [""]
     resources: ["persistentvolumes"]
-    verbs: ["get", "list", "watch", "update"]
+    verbs: ["get", "list", "watch", "update", "patch"]
```

Comment:
did you test the helm install /kustomize command? like if I have helm installed a prevoius version them helm install this version, i would assume that helm is smart enough to delete/recreate this object.